addons/website_slides/static/src/js/slides_course_quiz_question_form.js

```diff
@@ -123,6 +125,18 @@ var QuestionFormWidget = publicWidget.Widget.extend({
         });
     },
 
+    /**
+     * Handler when a user clicks on the 'Save & New' button.
+     * @param ev
+     * @private
+     */
+    async _validateAndCreateNewQuestion() {
+        await this._createOrUpdateQuestion({ update: false });
```

Comment:
I don't think we need to pass `update: false ` here. Can you please check without passing this?

Reply:
Sorry to say, but not passing `update: false` would result in an error on clicking the `Save & New` button. In the code snippet I have attached, the line `else if (options.update)` is the reason we will get an error if we do not pass it.

Also, passing the value as false makes sense, as we are not updating the existing question but creating a new one here. However if you wish to know the implementation of the `update` argument, you can have a look at the `_validateQuestion()` method (above `_validateAndCreateNewQuestion()`).

![image](https://github.com/odoo/odoo/assets/124355645/596afcf8-c99b-4bba-808b-335d0b0b1651)

Reply:
@shut-odoo Ok so for that can't we do something like `"options?.update"`? So in future, If someone has the same scenario then even they don't need to pass statically.

Reply:
@ibo-odoo yes we can do that :slightly_smiling_face: 
I have updated the code :+1: